code/modules/antagonists/blob/blobstrains/regenerative_materia.dm

```diff
@@ -26,9 +26,9 @@
 		exposed_mob.reagents.add_reagent(/datum/reagent/toxin/spore, 0.2*reac_volume)
 	exposed_mob.apply_damage(0.7*reac_volume, TOX)
 
-/datum/reagent/blob/regenerative_materia/on_mob_life(mob/living/carbon/metabolizer, seconds_per_tick)
+/datum/reagent/blob/regenerative_materia/on_mob_life(mob/living/carbon/metabolizer, seconds_per_tick, metabolized_volume)
 	. = ..()
-	if(metabolizer.adjust_tox_loss(1 * REM * seconds_per_tick, updating_health = FALSE))
+	if(metabolizer.adjust_tox_loss(0.5 * REM * metabolized_volume * seconds_per_tick, updating_health = FALSE))
```

Comment:
This whole pattern is kind of a mess; we should probably standardize a reagent multiplier that's fed in as like, an info packet that a reagent can then choose to manually act on, otherwise we're going to be chasing endless future bugs of people forgetting to multiply all 3 reagent multipliers or multiplying them in the wrong order or some shit.

Reply:
True. I'll think of something

Reply:
not sure about this one.

i don't like this not only bc of the sheer amt of work it'll produce downstream but i also just think a think a lot of chems will just stop working as intended because they have microdosing in mind.

and it makes things that naturally metabolize slowly much worse. i think this needs some very careful consideration